### DISCUSSION TITLE
How to get raw / unsmoothed glucose values to NS

### DISCUSSION THREAD

**Discussion by blaqone** [NightscoutFoundation/xDrip#2605] [Q&A]
Im gathering L2 glucose values via Juggluco and send them to xdrip. How can I disable any kind of value/graph smoothing and plain upload the raw values to NS?

**Comment by Navid200** [comment]
@psonnera @tzachi-dar

**Reply by psonnera** [reply]
xDrip doesn't upload raw to NS, only old Dexcom.

**Reply by blaqone** [reply]
I figured out how to do it via a oop2 - get raw to NS

**Reply by psonnera** [reply]
@blaqone oh, then it does work... amazing. Did you get it to display in NS with the plugin rawbg?

**Reply by blaqone** [reply]
No without any plugin.

**Reply by psonnera** [reply]
Can you post a piece of a screenshot to show how it comes in NS?

**Reply by blaqone** [reply]
![C37E6D53-B477-441F-B076-277253AFA1A1](https://user-images.githubusercontent.com/37814299/211195768-5b2b17aa-78e4-4dc1-9c53-99942898ffa4.jpeg)
![D5B921EA-6782-4B42-870A-61C626D76AF8](https://user-images.githubusercontent.com/37814299/211195770-a15a0db9-5382-4f87-a5ba-f6cb54122984.jpeg)
![8492B5D8-3F7D-42E8-8A6B-D128CCAB9A69](https://user-images.githubusercontent.com/37814299/211195772-a9bf16f5-8741-406a-81d8-aea4b9efc887.jpeg)
